apps/projects/app/components/Shared/BountyContextMenu.js

```diff
@@ -74,12 +74,14 @@ const BountyContextMenu = ({ issue }) => {
       )}
       {workStatus === 'review-applicants' && (
         <React.Fragment>
-          <Item onClick={() => requestAssignment(issue)}>
-            <IconFile color={`${theme.surfaceContent}`} />
-            <ActionLabel>
-              Submit application
-            </ActionLabel>
-          </Item>
+          {!pastDeadline && (
+            <Item onClick={() => requestAssignment(issue)}>
+              <IconFile color={`${theme.surfaceContent}`} />
+              <ActionLabel>
+                Submit Application
```

Comment:
I believe we want to keep this sentence-case. See #1682 & #1669

Reply:
Done